types/chrome/index.d.ts

```diff
@@ -3744,6 +3744,14 @@ declare namespace chrome.idle {
      * @param intervalInSeconds Threshold, in seconds, used to determine when the system is in an idle state.
      */
     export function setDetectionInterval(intervalInSeconds: number): void;
+    /**
+     * Gets the time, in seconds, it takes until the screen is locked automatically while idle. Returns a zero duration if the screen is never locked automatically.
+     * @requires(CrOS) Currently supported on Chrome OS only.
```

Comment:
I haven't seen `@requires(CrOS)` before, you sure about that?

Reply:
I updated it